packages/@react-spectrum/icon/src/Illustration.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps, DOMProps, StyleProps} from '@react-types/shared';
+import {classNames, useSlotProps, useStyleProps} from '@react-spectrum/utils';
+import {filterDOMProps} from '@react-aria/utils';
+import React, {ReactElement} from 'react';
+import styles from '@adobe/spectrum-css-temp/components/icon/vars.css';
+
+export interface IllustrationProps extends DOMProps, AriaLabelingProps, StyleProps {
+  /**
+   * A screen reader only label for the Illustration.
+   */
+  'aria-label'?: string,
+  /**
+   * The content to display. Should be an SVG.
+   */
+  children: ReactElement,
+  /**
+   * A slot to place the illustration in.
+   * @default 'illustration'
+   */
+  slot?: string,
+  /**
+   * Indicates whether the element is exposed to an accessibility API.
+   */
+  'aria-hidden'?: boolean | 'false' | 'true'
+}
+
+export type IllustrationPropsWithoutChildren = Omit<IllustrationProps, 'children'>;
+
+/**
+ * Wrapper component for illustrations. Use this to wrap your svg element for a custom illustration.
+ */
+export function Illustration(props: IllustrationProps) {
```

Comment:
Should I wrap our existing illustrations with this? Not sure if it'd be breaking since the props (may?) be different.

Reply:
illustrations have getIllustrationProps https://github.com/adobe/react-spectrum/blob/main/packages/%40spectrum-icons/illustrations/src/utils.ts
which appear to do some similar things that this does (guessing that was intentional)

so we could remove that function if we wrapped them with this, they'd gain the illustration slot, which is what IllustratedMessage already has, meaning that there's a potential for collision there

any props sent to the IllustratedMessage on the slot, would also be sent to the illustration below. We could explicitly clear the slots though (parents slots get merged in SlotProvider)

Reply:
Updated all illustrations to use the wrapper component, and cleared slots in IllustratedMessage